source/main/assets/export.tex

```diff
@@ -166,6 +166,15 @@
 \providecommand{\tightlist}{%
   \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
 
+$if(csl-refs)$
+\newlength{\cslhangindent}
+\setlength{\cslhangindent}{1.5em}
+\newenvironment{cslreferences}%
+{$if(csl-hanging-indent)$\setlength{\parindent}{0pt}%
+\everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces$endif$}%
+{\par}
+$endif$
+
```

Comment:
Did you check whether this code is also backwards-compatible to pre-2.8-versions of Pandoc?

Reply:
See the main PR message, the OP says it was tested on 2.7.3. From experience in another context I'm pretty sure this looks right.

Reply:
Oh damn it, I'm not completely awake today. Sorry, my mistake and thanks @alerque for pointing me to it! :)